src/components/Checkbox/Checkbox.js

```diff
@@ -29,7 +30,7 @@ const labelBaseStyles = ({ theme }) => css`
     content: '';
     display: block;
     position: absolute;
-    top: 50%;
+    top: 12px;
```

Comment:
Shouldn't this comes from theme?

Reply:
Technically, it's just half the line height...which also comes from the theme, so yes, you're absolutely right.